components/admin_console/group_settings/group_details/group_profile_and_settings.test.jsx

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import {GroupProfileAndSettings} from './group_profile_and_settings.jsx';
+
+describe('components/admin_console/group_settings/group_details/GroupProfileAndSettings', () => {
+    test('should match snapshot, with toggle off', () => {
+        const wrapper = shallow(
+            <GroupProfileAndSettings
+                displayname='GroupProfileAndSettings'
+                name='GroupProfileAndSettings'
+                allowReference={false}
+                onToggle={jest.fn()}
+            />);
+        expect(wrapper).toMatchSnapshot();
```

Comment:
nit: Would love to see explicit assertions vs. generic snapshot matches: I've historically found the snapshots too easy to change without noticing.